### PR TITLE
St feature lesson library screen

### PR DESCRIPTION
**Pull request by saveentoor**

### CODE REVIEW

File: client/js/screens/Lessons/LessonLibrary.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import { View, Text, TouchableOpacity, ScrollView } from "react-native";
+import { styles } from "./styles";
+import { SearchBar } from "react-native-elements";
+import TopicList from "../../components/TopicList";
+import topics from "../../helpers/topics";
+
+class LessonLibraryContainer extends Component {
+  static navigationOptions = {
+    title: "Libray"
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/LessonLibrary.js]: Correct the spellings of library.

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
+import React, { Component } from "react";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+
+class SurveyDifficultyContainer extends Component {
+  static navigationOptions = {
+    title: "LESSONS"
+  };
+  render() {
+    return (
+      <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file.

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        <Text
+          style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+            fontSize: 18
+          }}
+        >
+          What's your level
+          {"\n"}
+          of financial knowledge?
+        </Text>
+
+        <View
+          style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+            height: 220,
+            justifyContent: "center",
+            alignItems: "center",
+            borderRadius: 10,
+            marginBottom: 20,
+            fontFamily: "Quicksand"
+          }}
+        >
+          <View
+            style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+          />
+          <Image
+            style={{
+              width: 100,
+              height: 150
+            }}
+            source={require("../../../assets/images/illustrations/PNG/Tree_oranges.png")}
+          />
+          <TouchableOpacity
+            style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+              style={{
+                color: "white"
+              }}
+            >
+              Beginner
+            </Text>
+          </TouchableOpacity>
+        </View>
+        <View
+          style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+            backgroundColor: "#59c9c9",
+            width: 270,
+            height: 220,
+            justifyContent: "center",
+            alignItems: "center",
+            borderRadius: 10
+          }}
+        >
+          <View
+            style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+          />
+          <Image
+            style={{
+              width: 100,
+              height: 150
+            }}
+            source={require("../../../assets/images/illustrations/PNG/Tree_oranges.png")}
+          />
+          <TouchableOpacity
+            style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+            >
+              Advanced
+            </Text>
+          </TouchableOpacity>
+        </View>
+
+        <View>
+          <TouchableOpacity>
+            <Text
+              style={{
```
Comment (iyuvraajsingh) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Put it into styles file

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]
Good Work!

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js (hunk 3 above)
Comment (saveentoor) [reply client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: I didn't rebase, it is already in develop

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js (hunk 4 above)
Comment (saveentoor) [reply client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: I didn't rebase, it is already in develop

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js (hunk 6 above)
Comment (saveentoor) [reply client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: I didn't rebase, it is already in develop

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js (hunk 8 above)
Comment (saveentoor) [reply client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: I didn't rebase, it is already in develop

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js (hunk 10 above)
Comment (saveentoor) [reply client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: I didn't rebase, it is already in develop

File: client/js/screens/Lessons/LessonLibrary.js [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+      tabIndex: 0
+    };
+  }
+  updateSearch = search => {
+    this.setState({ search });
+  };
+
+  render() {
+    const { search } = this.state;
+    console.log(search);
```
Comment (siddharthparmar7) [inline client/js/screens/Lessons/LessonLibrary.js]: Please remove this console log.

File: client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+            >
+              skip
+            </Text>
+          </TouchableOpacity>
+        </View>
+      </View>
+    );
+  }
+}
+
```
Comment (siddharthparmar7) [inline client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js]: Please move all your styles to styles.js file

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
We do not use react-native elements. Please remove them and use react native components. Thank you. Please make the changes.

**Pull request closed without merging**